### ISSUE TITLE
[CRASH] notification and recent apps dont work without accessibility enabled that crashes the launcher

### ISSUE THREAD

**Issue by changatan** [Elnix90/Dragon-Launcher#387]
### Requirements

- [x] I checked for duplicate issues in both opened and closed
- [x] I updated the app to the latest version
- [ ] I updated the app to the latest beta version to test (check both if you did so)
- [ ] I haven't read those requirements

### Description

I successfully updated to the latest fdroid build 4.0. nest migrated with some sizing differences negligible. Recent apps and notification gestures require accessibility so it can display over apps it says.

### Steps to Reproduce

Go to settings enable accessibility on dragon launcher and it immediately crashes and doesn't work again till I disable accessibility. Making gesture for notifications and recent apps unusable for me.

### Device and version

Lenovo idea tab android 17
Pixel 9 grapheneOS android 17 both devices same issue

### Logs

```shell
java.lang.RuntimeException: Unable to create service org.elnix.dragonlauncher.services.system.SystemControlService: java.lang.ClassNotFoundException: Didn't find class "org.elnix.dragonlauncher.services.system.SystemControlService" on path: DexPathList[[zip file "/data/app/~~LxsHT9_HldeSXYwXoxEqog==/org.elnix.dragonlauncher-JwwzKWSaNDh6HXf6khm1_w==/base.apk"],nativeLibraryDirectories=[/data/app/~~LxsHT9_HldeSXYwXoxEqog==/org.elnix.dragonlauncher-JwwzKWSaNDh6HXf6khm1_w==/lib/arm64, /data/app/~~LxsHT9_HldeSXYwXoxEqog==/org.elnix.dragonlauncher-JwwzKWSaNDh6HXf6khm1_w==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
	at android.app.ActivityThread.handleCreateService(ActivityThread.java:5760)
	at android.app.ActivityThread.-$$Nest$mhandleCreateService(ActivityThread.java:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2817)
	at android.os.Handler.dispatchMessage(Handler.java:120)
	at android.os.Looper.loopOnce(Looper.java:249)
	at android.os.Looper.loop(Looper.java:339)
	at android.app.ActivityThread.main(ActivityThread.java:9785)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:618)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1022)
Caused by: java.lang.ClassNotFoundException: Didn't find class "org.elnix.dragonlauncher.services.system.SystemControlService" on path: DexPathList[[zip file "/data/app/~~LxsHT9_HldeSXYwXoxEqog==/org.elnix.dragonlauncher-JwwzKWSaNDh6HXf6khm1_w==/base.apk"],nativeLibraryDirectories=[/data/app/~~LxsHT9_HldeSXYwXoxEqog==/org.elnix.dragonlauncher-JwwzKWSaNDh6HXf6khm1_w==/lib/arm64, /data/app/~~LxsHT9_HldeSXYwXoxEqog==/org.elnix.dragonlauncher-JwwzKWSaNDh6HXf6khm1_w==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:644)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:580)
	at android.app.AppComponentFactory.instantiateService(AppComponentFactory.java:129)
	at android.app.ActivityThread.handleCreateService(ActivityThread.java:5719)
	... 9 more
```

### Additional Context

Thank you very much for your hard work I really enjoy your launcher.

**Comment by Elnix90** [comment]
Should be fixed in 4.0.1 and later, have you updated?

**Comment by Elnix90** [comment]
Well nevermind, it does not crash anymore but doesn't work

**Comment by changatan** [comment]
> Well nevermind, it does not crash anymore but doesn't work

It works now. I just had to allow restricted settings. I downloaded the apk. I didn't see the 4.0.1 update via fdroid. But I also don't know how to resync the repo on it manually

**Comment by Elnix90** [comment]
Yes, my bad it works.

Also you can use Obtainium to auto update as soon as github publishes a release
